db/people.yml

```diff
@@ -1955,6 +1955,7 @@ iadnan:
     - Ibrahim Adnan
     - Ibrahim, A
     - Ibrahim ALIbrahim
+  avatar: false
```

Comment:
this needs to be a url.  By default we set it to your image on your github account.   Then you can decide what to put there and it will show up on our group website.

Reply:
Done!